src/routes/index.js

```diff
@@ -17,54 +17,67 @@ const routes = {
   children: [
     {
       path: '/',
-      load: () => import(/* webpackChunkName: 'frontpage' */ './frontpage'),
+      load: () => import(/* webpackChunkName: 'frontpage' */ './frontpage')
     },
     {
-      path: '/4e94afa38918c6f2dcc12fd8a04d3972',
```

Comment:
We have to keep the submission form online for parties to submit their opinions. Can you make sure you add the old test again under this path and under /pr as well so that all the links we have sent out to the parties wont die all of sudden?

Reply:
Absolutely!